dsc/workflows/opencourseware.py

```diff
@@ -76,11 +75,10 @@ def item_metadata_iter(self) -> Iterator[dict[str, Any]]:
         for file in s3_client.files_iter(
             bucket=self.s3_bucket, prefix=self.batch_path, file_type=".zip"
         ):
-            zip_file = f"s3://{self.s3_bucket}/{file}"
-            item_identifier = file.split("/")[-1].removesuffix(".zip")
+            item_identifier = self.parse_item_identifier(file)
             yield {
-                "item_identifier": item_identifier,
-                **self._extract_metadata_from_zip_file(zip_file, item_identifier),
+                "item_identifier": self.parse_item_identifier(file),
```

Comment:
Sorry to nit-pick, but I think you have `item_identifier` in hand a couple of lines up, can this just be used here?

Reply:
Good catch! Fixed.